### PR TITLE
Make quotations in markdown consistent

### PR DESCRIPTION
**Pull request by cassiewallace**

### CODE REVIEW

**Comment by levino** [comment]
Good contribution. I recommend also making the markdown linting more strict and check for the quotes to be consistent in the future (allow only double quotes). Do you feel like adding this @cassiewallace ?

**Review by levino — COMMENTED** [review]
This is helpful but should be enforced in the future by markdown linting.

**Comment by cassiewallace** [comment]
Ack, will do!

**Comment by ericwindmill** [comment]
This is actually really important. In past versions of Gatsby (any maybe current), GraphQl queries will fail if there are descrepencies there. Thanks for catching.

**Comment by ericwindmill** [comment]
@cassiewallace This may be ingornance, but what exactly is "remark-lint-link-title-style"?

**Comment by cassiewallace** [comment]
@ericwindmill 

It's a package that enforces consistency in link and definition titles ([NPM link](https://www.npmjs.com/package/remark-lint-link-title-style)). It is included in some other MD lint presets, such as `remark-preset-lint-consistent`, but not `remark-preset-lint-recommended` which is already used in `gatsby-starter-docs`.

Alternatively, all of the `consistent` preset could be included here instead, but I chose to use just the part which was called out for this particular issue.

**Comment by levino** [comment]
The linting plugin does not work for our scenario. It will only check the titles in links, not in the frontmatter entries. See also [this issue](https://github.com/remarkjs/ideas/issues/3).

As a side note:

- Even if it worked, it should be a dev dependency, because we do not need it at runtime.
- Frontmatter allows also to not wrap things in quotes at all, maybe that would be even better.

As I do not have write access to this branch: @cassiewallace please remove the commit with the plugin and force push to the branch so the history does not get too cluttered. I will then approve and merge the PR.

File: package.json [hunk 1]
```diff
@@ -74,7 +75,8 @@
   },
   "remarkConfig": {
     "plugins": [
-      "remark-preset-lint-recommended"
+      "remark-preset-lint-recommended",
+      "remark-lint-link-title-style"
```
Comment (levino) [inline package.json]: I gave it a spin by setting one title to use single quotes and then running `npm run lint`. It did not throw an error, thus the plugin does not work for frontmatter.

**Comment by cassiewallace** [comment]
Done!

**Review by levino — APPROVED** [review]
Mucho ado... ;) Thanks.

**Pull request merged**